packages/iTwinUI-react/src/core/TimePicker/TimePicker.tsx

```diff
@@ -136,6 +183,32 @@ export const TimePicker = (props: TimePickerProps): JSX.Element => {
     secondRenderer = (date: Date) =>
       date.getSeconds().toLocaleString(undefined, { minimumIntegerDigits: 2 }),
     meridiemRenderer = (meridiem: MeridiemType) => meridiem,
+    useCombinedRenderer = false,
+    combinedRenderer = (date: Date, precision: Precision) => {
+      let dateString = '';
+      switch (precision) {
+        case 'seconds':
+          dateString =
+            ':' +
+            date
+              .getSeconds()
+              .toLocaleString(undefined, { minimumIntegerDigits: 2 });
```

Comment:
We leave that to the consumers of iTwinUI. So they could copy the code that I have there and insert info about the locale.

Reply:
Ohh, sounds good. I was just confirming like if locales mattered in the case of time as it does in the case of date. Also I wanted to confirm whether it would be better to pick any locale for the example that the user can edit to their needs. E.g. like the `DatePicker`'s [`Localized` story](https://itwin.github.io/iTwinUI-react/?path=/story/core-datepicker--localized)